dsa-23au/SoftwareSetup.md

```diff
@@ -106,7 +106,10 @@ Next to the "GitHub" integration, click on the three dots and choose "Manager th
 
 ### 7. Set your Git Config Email
 
-Run the `git-config.sh` script you set in the `GitHub` section above.
+Go to your `gitpod.io` environment variable settings at https://gitpod.io/user/variables.
+On this page add two new variables with the keys being
+`GIT_AUTHOR_EMAIL` and `GIT_COMMITER_EMAIL` and value being your github privacy email address. You will need to set a scope for this environment variables.
+`*/*` can be used if you want these emails to be applied to all workspaces opened in gitpod,or they could be set to `TheEvergreenStateCollege/upper-division-cs`. 
```

Comment:
For the last step, I think we can just recommend the scope as just for `theevergreenstatecollege/upper-division-cs`